apps/apps.humanatlas.io/src/app/pages/api/api.component.html

```diff
@@ -0,0 +1,57 @@
+<section class="splash">
+  <div class="video">
+    <video
```

Comment:
@LibbyUX Can we convert this video into a gif? Automatic playback doesn't always work

Reply:
@saaivignesh20 Please replace the video element using the gif Libby uploaded as a background image